mlrun/model_monitoring/controller.py

```diff
@@ -572,6 +595,16 @@ def model_endpoint_process(
 
                 endpoint_mode = mm_constants.EndpointMode.REAL_TIME
 
+                warnings.warn(
+                    "Analyzing batch model endpoints with real time processing events is deprecated. "
+                    "Please run serving as a job to invoke and analyze offline batch model endpoints.",
```

Comment:
also, we are not using please in errors
```suggestion
                    "Analyzing batch model endpoints with real time processing events is deprecated in 1.10.0 and will be removed in 1.12.0. "
                    "Use job-based serving to invoke and analyze offline batch model endpoints instead.",
```